src/main/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravService.kt

```diff
@@ -31,6 +31,11 @@ class AktivitetskravService(
         )
     }
 
+    fun createAktivitetskrav(personIdent: PersonIdent, previousAktivitetskrav: UUID?): PAktivitetskrav {
```

Comment:
Burde vi kalle parameteren `previousAktivitetskravUuid`?

Reply:
Det kan vi godt! Jeg er litt undecided selv på når man bør legge ved type i variabelnavnet🤔 Her finnes man jo på en måte den informasjonen om man kikker på typen. På samme måte som at man ikke lager variabelnavn med `…List` postfix.
Men i dette tilfellet er jeg enig i at det blir tydeligere om vi legger til `…Uuid` 🙏

Reply:
Ja, enig i at det ofte ikke trengs når det er gitt av typen, men har kan det virke som man har et aktivitetskrav-objekt så da er det fint med uuid :+1:

Reply:
Er det bedre med `previousAktivitetskravKey` eller noe tilsvarende da? Forklare hva det er domenemessig, heller enn teknisk 🤔

Reply:
Godt forslag, men lurer på om jeg da stemmer for å bare bruke `Uuid` 😊

Reply:
Hvorfor det? 🤔

Reply:
Problemet mitt er ikke at UUID er tvetydig, men at det ikke betyr noe som helst 🙃 Greit nok at `key` ikke er riktig begrep, men jeg tenker man må kunne finne et domeneord som beskriver hva dette er. Hvis man ikke finner et domeneord som er riktig, er det et tegn på at man gjør noe rart 🤷‍♂️